packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -519,9 +533,30 @@ class CupertinoPopupSurface extends StatelessWidget {
   /// Creates an iOS-style rounded rectangle popup surface.
   const CupertinoPopupSurface({
     super.key,
+    this.blurSigma = defaultBlurSigma,
     this.isSurfacePainted = true,
-    this.child,
-  });
+    this.isVibrancePainted = true,
+    required this.child,
+  }) : assert(blurSigma >= 0);
+
+  /// The strength of the gaussian blur applied to the area beneath this
+  /// surface.
+  ///
+  /// Defaults to [defaultBlurSigma]. Setting [blurSigma] to 0 will remove the
+  /// blur filter.
+  final double blurSigma;
+
+  /// Whether or not a [ColorFilter] should be applied beneath this surface.
```

Comment:
nit: specify the natural / purpose of the ColorFilter? It's only for adjusting the saturation I think?

Reply:
Added